index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alternative sports</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600&family=Roboto:ital,wght@0,100;0,400;0,500;0,700;0,900;1,100;1,400;1,500&display=swap');
+        </style>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+<!--Header section-->
+
+<!--Main page-->
+
+<section class="home" id="home">
+    <header>
+        <img src="Images/Earth.png" class="logo">         
+        
+        <nav class="navbar">
+            
+            <ul class="nav-menu">
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Home</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#article" class="nav-link">Articles</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#daily" class="nav-link">Sport of the day</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#footer" class="nav-link">Contact</a>
+                </li>
+        </ul>
+        <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+              </button>
+        </div>
+    </nav>
+
+</header>
+    
+    <video src="Video1.mp4" muted loop autoplay controls playsinline></video>
+    <div class="overlay"></div>
+
+      
+    <div class="home-text">
+        <h1>Alternative sports</h1>
+        <p>Latest news within the field of alternative sports<br>
+        Debitis veritatis voluptate officiis vero!<br></p>
+
+
+    <a href="#daily">Sport of the day</a>
+    <a href="#article" class="btn">Latest news</a>
+</div>
+</section>
+
+<!--Main page ends-->
+
+<!--Article page-->
+
+<section class="article" id="article">
+
+
+  <h1 class="heading">Latest news in the world of <span>Alternative sports</span></h1>
+
+  <div class="box-container">
+
+        
+  <div class="box">
+            <img src="Images/Latest-8.jpeg">
```

Comment:
could the naming of both folders and files be named with lowercase letters only, to not mix it up with javaScript naming? can be wrong here though!